index.ejs

```diff
@@ -12,25 +12,14 @@
   <div id="app">
     <p class="loading">Loading app...</p>
   </div>
+  
+  <!-- Zendesk -->
```

Comment:
Why add theses two lines here ?
This Zendesk comment has no meaning here, it is already 2 lines after.

Reply:
Bad copy paste...